sklearn/decomposition/_truncated_svd.py

```diff
@@ -115,9 +115,9 @@ class to data once, then keep the instance around to do transformations.
 
     References
     ----------
-    Finding structure with randomness: Stochastic algorithms for constructing
-    approximate matrix decompositions
-    Halko, et al., 2009 (arXiv:909) https://arxiv.org/pdf/0909.4061.pdf
+    :arxiv: `Halko, et al. (2009) "Finding structure with randomness:
```

Comment:
The `doc-min-dependencies` CI job is broken with the following message:

```
/home/circleci/project/sklearn/decomposition/_truncated_svd.py:docstring of sklearn.decomposition._truncated_svd.TruncatedSVD:109: WARNING: Field list ends without a blank line; unexpected unindent.'
```


I am not 100% sure but maybe this is caused by this unexpected extra blank space?

```suggestion
    :arxiv:`Halko, et al. (2009) "Finding structure with randomness:
```

The `doc` CI build is broken for a reason unrelated to this PR, I think.